doc/lightning-multifundchannel.7.md

```diff
@@ -0,0 +1,127 @@
+lightning-multifundchannel -- Command for establishing many lightning channels
+==============================================================================
+
+SYNOPSIS
+--------
+
+**multifundchannel** *destinations* \[*feerate*\] \[*minconf*\] \[*utxos*\]
+
+DESCRIPTION
+-----------
+
+The **multifundchannel** RPC command opens multiple payment channels
+with nodes by committing a single funding transaction to the blockchain
+that is shared by all channels.
+
+If not already connected, **multifundchannel** will automatically attempt
+to connect; you may provide a *@host:port* hint appended to the node ID
+so that c-lightning can learn how to connect to the node;
+see lightning-connect(7).
+
+Once the transaction is confirmed, normal channel operations may begin.
+Readiness is indicated by **listpeers** reporting a *state* of
+`CHANNELD_NORMAL` for the channel.
+
+*destinations* is an array of objects, with the fields:
+
+* *id* is the node ID, with an optional *@host:port* appended to it
+  in a manner understood by **connect**; see lightning-connect(7).
+  Each entry in the *destinations* array must have a unique node *id*.
+* *amount* is the amount in satoshis taken from the internal wallet
+  to fund the channel.
+  The string *all* can be used to specify all available funds
+  (or 16,777,215 satoshi if more is available and large channels were
+  not negotiated with the peer).
+  Otherwise it is in satoshi precision; it can be
+   a whole number,
+   a whole number ending in *sat*,
+   a whole number ending in *000msat*, or
+   a number with 1 to 8 decimal places ending in *btc*.
+  The value cannot be less than the dust limit, currently 546 satoshi
+  as of this writing, nor more than 16,777,215 satoshi
+  (unless large channels were negotiated with the peer).
+* *announce* is an optional flag that indicates whether to announce
+  the channel with this, default `true`.
+  If set to `false`, the channel is unpublished.
+* *push\_msat* is the amount of millisatoshis to outright give to the
+  node.
+  This is a gift to the peer, and you do not get a proof-of-payment
+  out of this.
+
+There must be at least one entry in *destinations*;
+it cannot be an empty array.
+
+*feerate* is an optional feerate used for the opening transaction and as
```

Comment:
i thought we revised feerate strings to be a wider set than just `urgent`, `normal`, `slow`, which are now considered deprecated?